### ISSUE TITLE
关于复现时在test集上指标低

### ISSUE THREAD

**Issue by lihuashengmax** [Haiyang-W/UniTR#11]
作者您好，我按照readme复现了unitr+lss版本，实验硬件为4*A40，由于每张A40显存为45G左右，即4张A40相当于8张3090，因此训练时能够保持总batchsize不变，但是提交test集的结果却比论文中的低了2个点，请问可能是什么原因呢？
![ClassIn_20231205154749](https://github.com/Haiyang-W/UniTR/assets/78634967/418875c0-4752-426a-b8c6-a279c7ef5a49)
![1701762708744](https://github.com/Haiyang-W/UniTR/assets/78634967/0d5f1586-e64f-41f3-8b6b-cfef4d9c8726)

**Comment by nnnth** [comment]
During evaluation on the test set, training should involve both the training and validation sets. Additionally, the ground truth  database should include the validation set. This approach is also adopted in BEVFusion and likely to yield an improvement of approximately 1~1.5 points. I suspect that's the reason.

**Comment by Haiyang-W** [comment]
Yes, this is a common trick used in Multi-modal 3D Detection, such as BEVFusion.

**Comment by lihuashengmax** [comment]
好的，感谢，我试一下

**Comment by LinuxCup** [comment]
@lihuashengmax ，您好！正如您上面所说，增加了val数据集性能有提升吗？ 谢谢了

**Comment by lihuashengmax** [comment]
> @lihuashengmax ，您好！正如您上面所说，增加了val数据集性能有提升吗？ 谢谢了

有一定提升，但仍未达到论文中在test集上的水平
